mediathread/assetmgr/views.py

```diff
@@ -1046,3 +1046,7 @@ def get(self, request):
         if len(notes) > 0:
             context = {'tags': TagResource().render_related(request, notes)}
         return self.render_to_json_response(context)
+
+
+class BookmarkletMigrationView(TemplateView):
```

Comment:
I put this in assetmgr because that's where all the bookmarklet-related templates are. I'll move the upgrade_bookmarklet view over here.